### DISCUSSION TITLE
CVE-2024-5535 fixed version in 3.0 stream

### DISCUSSION THREAD

**Discussion by IreshMM** [openssl/openssl#25014] [Q&A] [answered]
Is there a release fixing https://www.cve.org/CVERecord?id=CVE-2024-5535 on 3.0 stream?

**Comment by paulidale** [comment]
Yes, read the advisory you linked for details.

**Reply by IreshMM** [reply]
@paulidale Thanks for highlighting that. I hadn't noticed that.

**Comment by mattcaswell** [comment, marked as answer]
From the advisory:

https://openssl-library.org/news/secadv/20240627.txt

> OpenSSL 3.0 users should upgrade to OpenSSL 3.0.15 once it is released.

> Due to the low severity of this issue we are not issuing new releases of
> OpenSSL at this time. The fix will be included in the next releases when they
> become available. The fix is also available in commit e86ac436f0 (for 3.3),
> commit 99fb785a5f (for 3.2), commit 4ada436a19 (for 3.1) and commit cf6f91f612
> (for 3.0) in the OpenSSL git repository. It is available to premium support
> customers in commit b78ec0824d (for 1.1.1) and commit 99472514130 for (1.0.2).

3.0.15 has not yet been released.

**Comment by nikhil2611** [comment]
Is there any anticipated release date for 3.0.15 where CVE-2024-5535 is been fixed?

**Reply by t8m** [reply]
No date has been set yet.

**Reply by nikhil2611** [reply]
Thanks for the update.

**Comment by emilygaosalesforce** [comment]
I'm trying to understand the advisory:

> OpenSSL 1.0.2 users should upgrade to OpenSSL 1.0.2zk once it is released
(premium support customers only).

> The fix will be included in the next releases when they
become available.

> It is available to premium support
customers in commit b78ec0824d (for 1.1.1) and commit 99472514130 for (1.0.2).

Does this mean a patched version of 1.0.2 will come out at some point in time, available to non-premium support customers?

**Reply by t8m** [reply]
When it is released it will be available only to premium support customers.

**Reply by emilygaosalesforce** [reply]
Thank you, beyond that release date, will there be any intent to expand availability to beyond premium support customers?

**Reply by nhorman** [reply]
No

**Reply by emilygaosalesforce** [reply]
Understood, thank you!

**Comment by iyyapa** [comment]
@mattcaswell 
We are using  openssl-1.0.2h version. 
Are the CVE-2024-5535 affected for this version.

**Reply by t8m** [reply]
Yes, 1.0.2h is affected.

**Comment by sca007** [comment]
Hi , 
when is openssl 3.0.15 getting released.

Thanks

**Reply by t8m** [reply]
Today afternoon (UTC timezone).

**Reply by sca007** [reply]
Great, Thank you
